flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/BatchTableEnvironment.scala

```diff
@@ -296,4 +296,31 @@ abstract class BatchTableEnvironment(
           "This is a bug and should not happen. Please file an issue.")
     }
   }
+
+  final def tableFunctionToTable(udtf: String): Table = {
```

Comment:
To be honest I glad to call it `apply`.  and I'm not sure  is it right that we declare this method in env. Need a discussion.